SimFastTiming/FastTimingCommon/src/ETLElectronicsSim.cc

```diff
@@ -1,3 +1,5 @@
+#define EDM_ML_DEBUG
```

Comment:
@casarsa this is a remnant of debugging, it must be commented

Reply:
I've disabled the debug printouts.